### PR TITLE
Add timeout of 1.5 hours before resources become available again

### PR DESCRIPTION
**Pull request by dalfy**
The idea is to execute a look up on the table that will deallocate resources. 

The check is executed on all request to avoid having to run a thread. 

I need to run a couple of test before this can be deploy.

### CODE REVIEW

File: net_rm.py [hunk 1]
```diff
@@ -74,9 +74,25 @@ def custom503(error):
...
+                    resource[id],
+                    "Resource deallocated by resource manager"])
+    db.commit()
+
+
+
 @app.route('/status', methods=['GET'])
 def show_status():
     db = get_db()
+    revoke_timedout_allocation(db)
```
Comment (mrsixw) [inline net_rm.py]: Just call this in get_db()

File: net_rm.py [hunk 2]
```diff
@@ -74,9 +74,25 @@ def custom503(error):
     response.status_code = 503
     return response
 
+
+def revoke_timedout_allocation(db):
+    for resource in db.execute("SELECT id from resources WHERE datetime(now, 'unixepoch') - allocated_at > 5400"):
```
Comment (mrsixw) [inline net_rm.py]: can this not be "select id from resources where now() - allocated_at > 5400"??

File: templates/index.html [hunk 3]
```diff
@@ -34,6 +35,7 @@
             <td>{{ resource.allocated }}</td>
             <td>{{ resource.allocated_to_id }}</td>
             <td>{{ resource.allocated_to_address }}</td>
+            <td>{{ resource.allocated_at }}</td>
```
Comment (dalfy) [inline templates/index.html:38]: Should use moment.js to format the time and date.

Comment (mrsixw) [reply templates/index.html:38]: you want to make that change too??

Comment (dalfy) [reply templates/index.html:38]: Not now, that would mean integrating a javascript library in the app. 

Can't be asked

Comment (mrsixw) [reply templates/index.html:38]: OK, result now displays as an int? I think python probably has a fucntion to datify if so.

**Pull request merged**